requirements.txt

```diff
@@ -2,3 +2,5 @@ numpy~=1.15.0
 torch~=1.0.0
 torchvision~=0.2.0
 visdom~=0.1.0
+Pillow==6.1
+onnx==1.6.0
```

Comment:
`torch`, `torchvision` and `visdom` packages here are probably too old. Would you mind fixing them to the latest major.minor versions and update to something like `torch>=1.4.0`.